src/main/java/Duke.java

```diff
@@ -1,10 +1,139 @@
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileWriter;
+import java.util.ArrayList;
+
+
 public class Duke {
-    public static void main(String[] args) {
+
+    public static void printIntro(){
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("____________________________________________________________");
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void main(String[] args) throws UnknownCommandException,EmptyInputException,java.io.IOException{
+        printIntro();
+        Scanner in = new Scanner(System.in);
+        ArrayList<Task> list = new ArrayList<>();
+        int count=0;
+        //Task[] list = new Task[100];
+        String line;
+        File f = new File("data/Duke.txt");
+        if(f.exists()){
+            Scanner s = new Scanner(f);
+            while(s.hasNext()){
+                String[] temp = s.nextLine().split("\\|");
+                if(temp[0].equals("[T] ")){
+                    list.add(new ToDo(temp[2]));
+                    count++;
+                    if(temp[1].equals(" [V] ")){
+                        list.get(count-1).markAsDone();
+                    }
+                } else if(temp[0].equals("[D] ")){
+                    list.add(new Deadline(temp[2], temp[3]));
```

Comment:
Consider using temporary variables as labels so that others can understand what each fragment means